externs/browser/url.js

```diff
@@ -22,10 +22,20 @@
  * @author rdcronin@google.com (Devlin Cronin)
  */
 
+/**
+ * @typedef {IArrayLike<string>}
+ */
+var URLSearchParamsTupleType;
+
+/**
+ * @typedef {IArrayLike<!URLSearchParamsTupleType>|Iterable<!URLSearchParamsTupleType>}
+ */
+var URLSearchParamsTupleSequenceType;
+
 /**
  * @constructor
  * @implements {Iterable<!Array<string>>}
- * @param {(string|!URLSearchParams)=} init
+ * @param {(string|!URLSearchParamsTupleSequenceType|!Object<string,string>)=} init
```

Comment:
Could you add some comments explaining how `init` is used.
What does it mean for it to be a string?
What does it mean for it to be an `Object`?
What does it mean for it to be a sequence of tuples?